Bottom: 4613f1d7d2ddc668dac79c206c5e315c8aa5ec6e
Top:    54515d789e7b602f9a2b24ac06179a573d7923be
Author: scott snyder <sss@karma>
Date:   2018-08-26 20:24:30 -0400

Bump target version to 20.



---

diff --git a/ComicReader/project.properties b/ComicReader/project.properties
index ea89160..92b1625 100755
--- a/ComicReader/project.properties
+++ b/ComicReader/project.properties
@@ -8,4 +8,4 @@
 # project structure.
 
 # Project target.
-target=android-8
+target=android-20
